src/web_app_skeleton/config/database.cr.ecr

```diff
@@ -1,8 +1,8 @@
 database = "<%= crystal_project_name %>_#{Lucky::Env.name}"
 
 LuckyRecord::Repo.configure do
-  if Lucky::Env.production?
-    settings.url = ENV.fetch("DATABASE_URL")
+  if url = ENV["DATABASE_URL"]?
+    settings.url = url
   else
     settings.url = LuckyRecord::PostgresURL.build(
```

Comment:
I like this general idea, but I would like to keep it so that in production you are required to have a DATABASE_URL without it falling back to localhost. So instead, you could do something like this:

```crystal
if Lucky::Env.production?
  settings.url = ENV.fetch("DATABASE_URL")
else
  settings.url = ENV["DATABASE_URL"]? || LuckyRecord::PostgresURL.build(...)
end
```